ui/raidboss/data/05-shb/dungeon/matoyas_relict.js

```diff
@@ -0,0 +1,203 @@
+      condition: Conditions.caresAboutAOE(),
+      delaySeconds: 3,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Matoyas Porxie Huff and Puff',
+      netRegex: NetRegexes.startsUsing({ id: '5919', source: 'Mother Porxie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5919', source: 'Muttersau', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5919', source: 'mère porxie', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5919', source: 'マザーポークシー', capture: false }),
+      alarmText: (data, _, output) => output.standFront(),
+      outputStrings: {
+        standFront: {
+          en: 'Stand in front of Boss (anti-knockback not usable)',
+          ja: 'ボスの正面に (堅実魔効かない)',
+          cn: '站在Boss正面 (防击退无效)',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Porxie Meat Mallet',
+      netRegex: NetRegexes.startsUsing({ id: '5916', source: 'Mother Porxie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5916', source: 'Muttersau', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5916', source: 'mère porxie', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5916', source: 'マザーポークシー', capture: false }),
+      alarmText: (data, _, output) => output.awayFromAoe(),
+      outputStrings: {
+        awayFromAoe: {
+          en: 'Away From AoE',
+          ja: '反対側へ',
+          cn: '对面躲避坠落',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Porxie Barbeque',
+      netRegex: NetRegexes.ability({ id: '5916', source: 'Mother Porxie', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '5916', source: 'Muttersau', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '5916', source: 'mère porxie', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '5916', source: 'マザーポークシー', capture: false }),
+      delaySeconds: 5,
+      alarmText: (data, _, output) => output.awayFrom(),
+      outputStrings: {
+        awayFrom: {
+          en: 'Away From Boss',
+          ja: 'ボスに離れ',
+          cn: '远离Boss所在的行',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Porxie Sucked In',
+      netRegex: NetRegexes.gainsEffect({ effectId: '5916', capture: false }),
+      alarmText: (data, _, output) => output.runAway(),
+      outputStrings: {
+        runAway: {
+          en: 'RUN AWAY',
+          ja: 'ボスに離れ',
+          cn: '远离即死区',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Porxie Minced Meat',
+      netRegex: NetRegexes.startsUsing({ id: '5911', source: 'Mother Porxie' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5911', source: 'Muttersau' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5911', source: 'mère porxie' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5911', source: 'マザーポークシー' }),
+      condition: function(data, matches) {
+        return data.me === matches.target || data.role === 'healer';
+      },
```

Comment:
The tankbuster is magical
```suggestion
      condition: Conditions.caresAboutMagical(),
```